components/greetings/greetings.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./greetings.module.css";
+
+const date = new Date();
+const currentTime = date.getHours();
+
+let greetings;
+
+if (currentTime < 12) {
+  greetings = "Good Morning";
+} else if (currentTime < 18) {
+  greetings = "Good Afternoon";
+} else if (currentTime < 22) {
+  greetings = "Good Evening";
+} else {
+  greetings = "Good Night";
+}
```

Comment:
Move this into the function and into a `useEffect` to only calls this once the page has loaded

Reply:
Will open a further Pull Request for it, thanks!